src/Control/Optics/Linear.hs

```diff
@@ -1,3 +1,136 @@
+-- | This module provides linear optics
```

Comment:
I think these first sentences should end with a period. The same applies to other modules too.